doc/source/ray-core/examples/batch_training.ipynb

```diff
@@ -493,11 +489,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Loading data once into Ray object store\n",
+    "## Centralized data loading using Ray object store\n",
```

Comment:
Prefix this section with something along the lines of `[Optional]` or `[Optimization]` ?